package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "coffeeify",
-  "version": "2.1.0",
-  "description": "browserify plugin for coffee-script with support for mixed .js and .coffee files",
+  "version": "2.2.0",
+  "description": "browserify plugin for coffeescript with support for mixed .js and .coffee files",
   "main": "index.js",
   "dependencies": {
-    "coffee-script": "^1.12.0",
     "convert-source-map": "^1.3.0",
     "through2": "^2.0.0"
   },
+  "peerDependencies": {
+    "coffeescript": "*"
```

Comment:
Set the peer dependency version to `<=2` so we can handle it if you ever feel like breaking the compiler api :)